### PR TITLE
Initial SteamVR 0.9.16 support

### PR DESCRIPTION
**Pull request by d235j**
This is completely untested but it does compile. Please test before merging!

### CODE REVIEW

File: ValveStrCpy.h [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+//               Copyright Sensics, Inc. 2016.
+//  Distributed under the Boost Software License, Version 1.0.
+//     (See accompanying file LICENSE_1_0.txt or copy at
+//           http://www.boost.org/LICENSE_1_0.txt)
+#ifndef INCLUDED_ValveStrCpy_h_GUID_E8F022B2_8826_40B2_E94E_AA1F2A33ED74
```
Comment (toxicFork) [inline ValveStrCpy.h]: how do you generate these guids?

**Comment by godbyk** [comment]
And this is out of date already. There's yet another new release of OpenVR: <https://github.com/ValveSoftware/openvr/releases/tag/v0.9.17>

File: ValveStrCpy.h (hunk 1 above)
Comment (godbyk) [inline ValveStrCpy.h]: @toxicFork You can use the OSVR boilerplate generator: <http://tools.getosvr.org/boilerplate/>.

Comment (toxicFork) [inline ValveStrCpy.h]: Neat, thanks :)

**Comment by d235j** [comment]
Thanks to @jcp219 and @toxicFork this looks good to go! Should also fix #33 and maybe #31.

**Pull request merged**